src/main/java/seedu/address/model/job/Job.java

```diff
@@ -0,0 +1,95 @@
+package seedu.address.model.job;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.model.person.Name;
+
+/**
+ * Represents a Job in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Job {
+
+    // Data fields
+    private final Name jobTitle;
+    private final Name companyName;
+    private final Name industry;
```

Comment:
As we'll also need to add Jobs in a company object, this `Job` object seems to differ implementation-wise from the `Job` object that we may use for company objects, though they should probably be the same? For example, in a `Company` object, during its creation via a user command, the user can input multiple industries and jobs that a company has. This creates the `Company` object, along with its multiple jobs and industries. So in those `Job` objects in the `Company` object, will each of them have multiple industries tagged to them instead of just one industry as in this `Job` object? Also, since a `Company` stores `Job` objects, would this create a cyclical dependency if we change the `Name companyName` part to `Company company` in the future (I assume we will)? At this juncture, it seems that if `Job` is going to store Company and Industry, perhaps we may not even need `Job` objects? Perhaps we could simply use String (or Name) for jobs? This may also help to synchronize with how jobs will have to be represented in a `Company` object.

Reply:
Yes, I did thought of this while I was writing the code for this Job class. Yes, I think that we may need need a Job object and it can be represented as a `String` instead. Great catch on this!